backend/apps/readings/analysis.py

```diff
@@ -153,6 +153,50 @@ def most_common_response_by_question_and_context(all_responses, question, contex
     return max_response
 
 
+def repeated_prompt_words(responses):
+    """
+    Calculates frequencies of exact words from story prompt used in responses (disregards words
+    not in story prompt).
+    :param responses: list of words used in a set of responses
+    :return: A dictionary, resp_words. Key is a word from the prompt, value is its frequency
+    among
+    responses
+    """
+    # dictionary to store frequencies of words in responses
+    resp_words = {}
+
+    # cycling through responses
+    stop_words = ['a', 'and', 'the', 'of', 'an', 'for']
+    for word in responses:
+        if word in resp_words:
+            resp_words[word] += 1
+        elif word not in stop_words:
+            resp_words[word] = 1
+
+    # list of words in story
+    # TODO for longer story, would want to read it from csv file instead of hardcoding
+    story_vocab_list = ["for", "sale", "baby", "shoes", "never", "worn"]
```

Comment:
Let's stop hardcoding these vocab lists -- won't be useful moving forward. Maybe look at the actual text from the db model and remove stop words to generate a vocab list.